charts/prometheus/values.yaml

```diff
@@ -47,4 +47,4 @@ persistence:
 ports:
   http: 9090
 
-ingressNetworkPolicyExtraSelectors: []
```

Comment:
I know we don't like unrelated changes... but we want newlines, right?

Reply:
Yep